src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -358,6 +370,38 @@ public List<int> ConsensusTipChanged(ChainedHeader newConsensusTip)
         {
             this.logger.LogTrace("({0}:'{1}')", nameof(newConsensusTip), newConsensusTip);
 
+            ChainedHeader oldConsensusTip = this.GetConsensusTip();
+            ChainedHeader fork = newConsensusTip.FindFork(oldConsensusTip);
+            ChainedHeader currentHeader = newConsensusTip;
+
```

Comment:
It might be nice to have all three guys logged – the old consensus tip, the new one, and the fork.

Reply:
Fixed